source/VMelnalksnis.PaperlessDotNet/Documents/IDocumentClient.cs

```diff
@@ -48,6 +49,13 @@ public interface IDocumentClient
 	/// <returns>The document with the specified id if it exists; otherwise <see langword="null"/>.</returns>
 	Task<Document<TFields>?> Get<TFields>(int id, CancellationToken cancellationToken = default);
 
+	/// <summary>Downloads the file of the document.</summary>
+	/// <param name="id">The id of the document to download.</param>
+	/// <param name="original">Download the original document when an archived document is available.</param>
+	/// <param name="cancellationToken">A cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
+	/// <returns>The content of the document.</returns>
+	Task<DocumentContent> Download(int id, bool original, CancellationToken cancellationToken = default);
```

Comment:
Instead of a `bool` parameter, should rather just have to methods - `Download` and `DownloadOriginal`
```suggestion
	Task<DocumentContent> Download(int id, CancellationToken cancellationToken = default);
	
	Task<DocumentContent> DownloadOriginal(int id, CancellationToken cancellationToken = default);
```

Reply:
I added additional Orignal-methods for both the download of the document and the preview.